### ISSUE TITLE
PHP Errors in containers upon startup

### ISSUE THREAD

**Issue by aepod** [magento/magento-cloud-docker#128]
Because of the comments added to php ini files there is errors logged.

### Preconditions
1. On current develop branch

### Steps to reproduce
1. Bring up containers
2. check logs using docker-compose logs fpm

### Expected result
1. All offending lines would not show
```
[webuser@localhost ztgento]$ docker-compose logs fpm
/usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.6) or chardet (2.2.1) doesn't match a supported version!
  RequestsDependencyWarning)
Attaching to ztgento_fpm_1
fpm_1            | [16-Jan-2020 18:43:02] NOTICE: fpm is running, pid 1
fpm_1            | [16-Jan-2020 18:43:02] NOTICE: ready to handle connections

```

### Actual result
```
[webuser@localhost ztgento]$ docker-compose logs fpm
Attaching to ztgento_fpm_1
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | PHP:  syntax error, unexpected '!' in /usr/local/etc/php/conf.d/zz-magento.ini on line 2
fpm_1            | [16-Jan-2020 18:33:44] NOTICE: fpm is running, pid 1
fpm_1            | [16-Jan-2020 18:33:44] NOTICE: ready to handle connections

```

**Comment by andriyShevtsov** [comment]
Created internal task `MAGECLOUD-5142`

**Comment by swnsma** [comment]
Any updates?

**Comment by aepod** [comment]
This was merged and released in 1.1, should not be happening if you are up to date.
